File: app-android/app/src/main/java/com/health2u/data/remote/dto/EmergencyContactDto.kt
package com.health2u.data.remote.dto

import com.google.gson.annotations.SerializedName

data class EmergencyContactDto(
    @SerializedName("id") val id: String,
    @SerializedName("user_id") val userId: String,
    @SerializedName("name") val name: String,
    @SerializedName("relationship") val relationship: String,
    @SerializedName("phone") val phone: String,
    @SerializedName("email") val email: String?,
    @SerializedName("is_primary") val isPrimary: Boolean,
    @SerializedName("order") val order: Int
)

File: app-ios/Sources/Health2u/Data/Remote/DTOs/EmergencyContactDTO.swift
import Foundation

public struct EmergencyContactDTO: Codable, Equatable, Sendable {
    public let id: String
    public let userId: String
    public let name: String
    public let relationship: String
    public let phone: String
    public let email: String?
    public let isPrimary: Bool
    public let order: Int

    public init(
        id: String,
        userId: String,
        name: String,
        relationship: String,
        phone: String,
        email: String? = nil,
        isPrimary: Bool,
        order: Int
    ) {
        self.id = id
        self.userId = userId
        self.name = name
        self.relationship = relationship
        self.phone = phone
        self.email = email
        self.isPrimary = isPrimary
        self.order = order
    }
}
